File: Python developer/step04/4.1.01.py
'''

Ваша программа принимает целое число n. 
Напишите программу, определяющую что число находится в интервале от 5 включительно до 10 строго или от 101 строго до 200 включительно 
или иными словами n∈[5,10)∪(101,200]n∈[5,10)∪(101,200]. В качестве ответа выведите True если находится и False если не находится.

Sample Input 1:

5

Sample Output 1:

True

Sample Input 2:

10

Sample Output 2:

False

Sample Input 3:

7

Sample Output 3:

True

Sample Input 4:

101

Sample Output 4:

False

Sample Input 5:

200

Sample Output 5:

True

'''

a = int(input())
print(5<=a<10 or 101<a<=200)

File: Diving into JavaScript: For Beginners/step04/4.01/4.1.01.js
/*

Перед вами программа, которая должна выводить каждое число по порядку от 1 до заданного пользователем числа (включительно). 
Напишите вместо многоточия правильное условие.

Sample Input 1:

3

Sample Output 1:

1
2
3

Sample Input 2:

6

Sample Output 2:

1
2
3
4
5
6

Sample Input 3:

2

Sample Output 3:

1
2

*/

let targetNumber = Number(prompt("Введите число, до которого нужно считать:"));
let currentNumber = 1;

while (currentNumber <= targetNumber) {
  console.log(currentNumber);
  currentNumber++;
}
